File: Iniciante/1828.py
nTestes = int(input())

for i in range(1, nTestes+1):
  r, s = map(str, input().split())
  
  if (r == s):
    print("Caso #{}: De novo!".format(i))
  elif (r == 'tesoura' and s == 'papel' or r == 'tesoura' and s == 'lagarto'):
    print("Caso #{}: Bazinga!".format(i))
  elif (r == 'papel' and s == 'pedra' or r == 'papel' and s == 'Spock'):
    print("Caso #{}: Bazinga!".format(i))
  elif (r == 'pedra' and s == 'lagarto' or r == 'pedra' and s == 'tesoura'):
    print("Caso #{}: Bazinga!".format(i))
  elif (r == 'Spock' and s == 'tesoura' or r == 'Spock' and s == 'pedra'):
    print("Caso #{}: Bazinga!".format(i))
  elif (r == 'lagarto' and s == 'Spock' or r == 'lagarto' and s == 'papel'):
    print("Caso #{}: Bazinga!".format(i))
  else:
    print("Caso #{}: Raj trapaceou!".format(i))

File: Iniciante/1828.js
let nTestes = Number(lines.shift());

for (let i = 1; i <= nTestes; i++) {
  let [r, s] = lines.shift().split(' ');
  
  if (r === s) {
    console.log(`Caso #${[i]}: De novo!`);
  } else if (r == 'tesoura' && s == 'papel' || r == 'tesoura' && s == 'lagarto') {
    console.log(`Caso #${[i]}: Bazinga!`);
  } else if (r == 'papel' && s == 'pedra' || r == 'papel' && s == 'Spock') {
    console.log(`Caso #${[i]}: Bazinga!`);
  } else if (r == 'pedra' && s == 'lagarto' || r == 'pedra' && s == 'tesoura') {
    console.log(`Caso #${[i]}: Bazinga!`);
  } else if (r == 'Spock' && s == 'tesoura' || r == 'Spock' && s == 'pedra') {
    console.log(`Caso #${[i]}: Bazinga!`);
  } else if (r == 'lagarto' && s == 'Spock' || r == 'lagarto' && s == 'papel') {
    console.log(`Caso #${[i]}: Bazinga!`);
  } else {
    console.log(`Caso #${[i]}: Raj trapaceou!`);
  }
}
